docs/reference/predicates.md

```diff
@@ -128,6 +128,44 @@ Host(/^my-host-header\.example\.org$/)
 Host(/header\.example\.org$/)
 ```
 
+## Forwarded header predicates
+
+Uses standardized Forwarded header ([RFC 7239](https://tools.ietf.org/html/rfc7239))
+
+More info about the header: [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded)
+
+### ForwardedHost
+
+Regular expressions that the forwarded host header in the request must match.
+
+Parameters:
+
+* Host (regex)
+
+Examples:
+
+```
+ForwardedHost(/^my-host-header\.example\.org$/)
```

Comment:
These predicates are equivalent to 
* `HeaderRegexp("Forwarded", "host=example\.com(;|$)")`
* `HeaderRegexp("Forwarded", "proto=http(;|$)")`
from what I can tell.

Reply:
Do you think we shouldn't add the predicates? or do you think we should just reuse the ones that exist (even if with a tiny layer on top?)

Reply:
> Do you think we shouldn't add the predicates? 

It is more of a note for myself, you and reviewers. I am not a fan of regexp hacking and prefer dedicated thing but new predicates also add maintenance/compatibility cost which we have to weight against the benefit.

> Or are you suggesting we should just reuse the ones that exist (even if with a tiny layer on top?)

I am not sure we could reuse them as they are legacy predicates implemented differently say from `Source`.

Reply:
That's much harder than I thoug... If "basic" predicates were exposed like any other predicate, it should be easy to do something like this:
```
func (p *hostPredicateSpec) Create(args []interface{}) (routing.Predicate, error) {
    spec := headerRegexp.New()
    return spec.Create("Forwarded", fmt.Sprintf("host=example\.com(;|$)", args[0])
}
```

Reply:
This is an interesting case and why we should move the legacy predicates (I would not call them "basic").
IIRC there is no reason for non path related predicates to stay in the eskip package besides breaking compatibility.
Do we see a solution without a breaking change?
I am not really sure if someone had a use of single eskip package and use the legacy predicates from there as exposed in `eskip.Route`

Reply:
`legacy` is definitely a better way to describe them than `basic` :)